.github/actions/prepare-env/action.yml

```diff
@@ -6,6 +6,9 @@ name: Prepare environment
 description: Install dependencies and prepare environment needed for OpenTitan
 
 inputs:
+  service_account_json:
+    description: Service account JSON for Google Cloud access
+    default: ''
```

Comment:
Can this default to `'${{ secrets.BAZEL_CACHE_CREDS }}'`? We're only ever going to pass this value from callers

Reply:
Actions can't have access to secrets directly, only workflows can.

Reply:
I guess that makes sense